City-of-Helsinki/berth-reservations-admin
# VEN-580, VEN-582 | Add berth details sub component and missing columns to individual harbor page

## Description :sparkles:

- Adds an expandable details card to each berth in the individual harbor page's table.
- Adds missing columns to the berth table.

## Issues :bug:

### Closes :no_good_woman:

VEN-582

### Related :handshake:

## Testing :alembic:

### Automated tests :gear:️

### Manual testing :construction_worker_man:

* Open an harbor's page.
* Toggle the subcomponent for a berth row.
* The subcomponent should display the info specified in designs.

## Screenshots :camera_flash:

## Additional notes :spiral_notepad:
